### PR TITLE
Docz external resolve fix

### PR DESCRIPTION
**Pull request by ejuo**
Fix for #783

### CODE REVIEW

**Comment by pedronauck** [comment]
Can you explain to me how this could improve or what this change? Because all dependencies and external are already not bundled.

File: core/docz-theme-default/rollup.config.js [hunk 1]
```diff
@@ -1,12 +1,5 @@
 import { config } from 'docz-rollup'
 
 export default config({
-  input: 'src/index.tsx',
-  external: id =>
-    !id.startsWith('\0') &&
-    !id.startsWith('.') &&
-    !id.startsWith(process.platform === 'win32' ? process.cwd() : '/') &&
-    !id.startsWith('@components') &&
```
Comment (pedronauck) [inline core/docz-theme-default/rollup.config.js]: And about this?

**Comment by ejuo** [comment]
> Can you explain to me how this could improve or what this changes? Because all dependencies and external are already not bundled.

This is an alternative (cross-platform) solution for searching external dependencies. Do you think the expression like !id.startsWith ('.') is safe?

**Comment by pedronauck** [comment]
I never have any problem with this, but we can keep both I think, we don't need to remove the default external configuration and the external configuration configured inside the modules, just add this plugin in the default plugins.

**Review by pedronauck — APPROVED** [review]

**Pull request merged**